core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/appRoutesModels.scala

```diff
@@ -39,6 +39,8 @@ final case class CreateAppRequest(kubernetesRuntimeConfig: Option[KubernetesRunt
                                   autodeleteEnabled: Option[Boolean]
 )
 
+final case class UpdateAppConfigRequest(autodeleteEnabled: Option[Boolean], autodeleteThreshold: Option[Int])
```

Comment:
I think it'll be helpful to lay out all the permutations here. And what assumptions/defaults we're making in the code.
Like if both are none, and DB has `autodeleteEnabled` being `true`, does that mean we're disabling auto delete?

Reply:
this is now explicitly laid out in validateUpdateAppConfigRequest()

Reply:
This conversation inspired this section of the design doc https://docs.google.com/document/d/1eWfY8PQoDKVOdIeREuEC9ObLhXlsTtG2ugLwuqov81k/edit#bookmark=id.n9sphs8lpm4l

Reply:
Up to you if you want to do it, but if we use a custom type for `autodeleteThreshold` (say `AutodeleteThreshold`), we can then define a decoder for it and verify it has to be positive in the decoder. This way, anywhere this type is used, it's guaranteed to be postive(let's say we use this custom type in `CreateAppRequest`)